kyuubi-common/src/main/scala/org/apache/kyuubi/engine/EngineType.scala

```diff
@@ -23,5 +23,5 @@ package org.apache.kyuubi.engine
 object EngineType extends Enumeration {
   type EngineType = Value
 
-  val SPARK_SQL, FLINK_SQL, TRINO, HIVE_SQL = Value
```

Comment:
we are buiding a doris connector only or common jdbc engine?

Reply:
yeah, it can be  common jdbc engine, but may need other config to identify type.

Reply:
using jdbc engine type, and add a config of kyuubi.engine.jdbc.type to identify jdbc type.